parameters.d/osg34-stashcache.yaml

```diff
@@ -0,0 +1,36 @@
+###################
+# OSG 3.4 tests for StashCache 1.0.2 (in osg-testing)
+# File format documention:
+# https://github.com/opensciencegrid/vm-test-runs#running-osg-test-in-vm-universe
+###################
+
+platforms:
+  - centos_7_x86_64
+  - sl_7_x86_64
+
+sources:
+  ###################
+  # Format:
+  # [<Github account>:<osg-test branch>;] <OSG ver>; <REPO 1, REPO 2...REPO N> [> <Update OSG ver>/<Update REPO 1, REPO 2...REPO N>]
+  # Example:
+  # Run osg-test (from 3.2-minefield) with packages from 3.2-release and 3.2-testing that are then upgraded to
+  # 3.3-testing and 3-3-upcoming-testing:
+  # 3.2; osg, osg-testing > 3.3/osg-testing, osg-upcoming-testing
+  ###################
+  - opensciencegrid:xcache-1.0.2; 3.4; osg-testing
```

Comment:
I'm not sure how we could do that since the package names changed; when I ask for 'stashcache-cache-server', 'stash-cache' provides that dependency, and then osg-test errors out because the literal package named stashcache-cache-server isn't installed.

Reply:
Basically another copy of https://github.com/opensciencegrid/vm-test-runs/blob/master/parameters.d/osg34-el7-only.yaml minus these sections:

- https://github.com/opensciencegrid/vm-test-runs/blob/master/parameters.d/osg34-el7-only.yaml#L22-L23
- https://github.com/opensciencegrid/vm-test-runs/blob/master/parameters.d/osg34-el7-only.yaml#L34-L48
- https://github.com/opensciencegrid/vm-test-runs/blob/master/parameters.d/osg34-el7-only.yaml#L56-L66

We just want to keep testing the current release of StashCache in the nightlies